build_tools/scripts/git/update_runtime_submodules.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+# Copyright 2022 The IREE Authors
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+#
+# Initialize a subset of submodules for runtime only.
+
+set -xeuo pipefail
+
+RUNTIME_SUBMODULES=(
```

Comment:
Can we avoid duplicating this please?

Reply:
Move them to `runtime_submodules.txt` and use it in both scripts.